server/routes/projectRoutes.js

```diff
@@ -11,7 +11,7 @@ const auth = require('../middleware/authMiddleware');
 
 router.get('/', getAllProjects); // Get all projects (public)
 router.get('/:projectId', getProject);
-//router.get('/:userId', getProjectsForUser); // Get all projects for a specific user
+router.get('/:userId/projects', getProjectsForUser); // Get all projects for a specific user
```

Comment:
thoughts on this route being ('/all/:userId')

Reply:
Makes sense. Will change it to /all/:userId